src/Pretty.hs

```diff
@@ -58,8 +62,21 @@ angles doc = text "⟨" <> align (group doc) <> "⟩"
 vec :: Doc ann -> Doc ann
 vec doc = text "[" <> align (group doc) <> "]"
 
+customLayoutOptions :: LayoutOptions
+customLayoutOptions = unsafePerformIO $ do
```

Comment:
Using `unsafePerformIO` is the most straightforward way to do it, but of course I expect that somebody will object and that I will have to thread a `LayoutOptions` variable everywhere.

Reply:
One concern is that the tests use `pretty`, so they might start failing depending on the terminal width. But I tried making my terminal very small and the tests still passed :shrug:

Reply:
Oh, that's because it doesn't work. `echo $COLUMNS` prints the right value in the terminal, but `lookupEnv "COLUMNS"` returns `Nothing`!